apps/admin/traefik2/demo/kustomization.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ../../base
+  - ../../base
+  - ./traefik-health-endpoint.yaml
```

Comment:
that a typo?

Reply:
Sorry @thomast1906  thought I'd reverted the changes to that file already, sorted that now